Source/WebKit/WebProcess/WebCoreSupport/WebContextMenuClient.cpp

```diff
@@ -94,11 +94,16 @@ void WebContextMenuClient::stopSpeaking()
 
 void WebContextMenuClient::showContextMenu()
 {
-    m_page->contextMenu().show();
+    protectedPage()->protectedContextMenu()->show();
```

Comment:
I think we may want to null check `m_page` instead of introducing `protectedPage()` here. Would look like:
```
if (RefPtr page = m_page.get())
    page->protectedContextMenu()->show();
```